### PR TITLE
Redwan dev

### PR DESCRIPTION
**Pull request by red1habibullah**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,6 @@ source 'https://rubygems.org'
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
 # Synchronize with https://pages.github.com/versions
-ruby '>=2.5.8'
+ruby '>=2.5.1'
```
Comment (aperloff) [inline Gemfile]: Why did you change the Gemfile, especially to lower it to a less recent version?

File: _episodes/02-CMSDataAnalysisSchoolPreExerciseSecondSet.md [hunk 2]
```diff
@@ -1,19 +1,363 @@
...
+
+`Exercises 8` and `Exercise 9` are using FWLite (**Frame Work Lite**). This is an interactive analysis tool integrated with the CMSSW EDM (Event Data Model) Framework. It allows you to automatically load the shared libraries defining CMSSW data formats and the tools provided, to easily access parts of the event in the EDM format within ROOT interactive sessions. It reads produced ROOT files, has full access to the class methods and there is no need to write full-blown framework modules. Thus having FWLite distribution locally on the desktop one can do CMS analysis outside the full CMSSW framework. In these two exercises, we will analyze the data stored in a MiniAOD sample using FWLite. We will loop over muons and make a Z mass peak.
+
+**We assume that having done the first set of pre-exercises by now, one is comfortable with logging onto** `cmslpc-sl7.fnal.gov` **and setting up the cms environment.**
+
+# Support
```
Comment (aperloff) [inline _episodes/02-CMSDataAnalysisSchoolPreExerciseSecondSet.md]: As you can see from the other exercises, we didn't include the Support, Support email, mattermost, or Obtain a ... sections. All of this is included in the setup directions. It's better to have one authoritative spot for all of those directions. Make sure what is in the setup.md document is good enough for you and then remove these sections from this document.

File: Gemfile (hunk 1 above)
Comment (red1habibullah) [reply Gemfile]: It was a temporary fix for me to look at the HTML, it seems the ruby version requirement was not working for my local Linux package, versions. I can fix it back in the next commit.
